src/controllers/observation-controller.ts

```diff
@@ -63,29 +72,18 @@ export default class ObservationController extends AbstractController {
       const langOrigin = LocaleOrigin[lang] ? LocaleOrigin[lang] : 'desc_eng';
 
       const paramsSearch = parsePageParams(req.query);
-      const paramsAggregation = parseWhereParams(req.query, req.user);
+      const paramsAggregation = parseWhereParams(req.user, req.query);
       const observations = await this.observations.findAndCount(Object.assign(paramsSearch, paramsAggregation));
 
       const content = observations[0].map(obs => {
```

Comment:
try to use or on letter like `o` or full key `observation` without inter-immediate variants, with one known and widely used shorhand for `reduce`: `accumulator` = > `acc | accum | others`